packages/sources/react/src/components/actions/VtmnLink/VtmnLink.tsx

```diff
@@ -17,6 +17,21 @@ export interface VtmnLinkProps extends React.ComponentPropsWithoutRef<'a'> {
    */
   standalone?: boolean;
 
+  /**
+   * The target attribute of the link.
+   * @type {string}
+   * @defaultValue undefined
+   * @requires
+   */
+  target: string;
+
+  /**
+   * The rel attribute of the link.
+   * @type {string}
+   * @defaultValue undefined
+   */
+  rel?: string;
+
```

Comment:
I'm not sure that we should rewrite the whole (or part of the) HTML link standard API here:
- User should be able to use any legal HTML link attribute on our `VtmnLink`. User should just write attributes and this attributes should be on the rendered link ;
- Optional HTML link attribute should not be made mandatory for no reason in our `VtmnLink` (with maybe some accessibility exception).

Meaning that `target` and `rel`, that are already part of the link api, should not be a component props.
Instead we should maybe just "intercept" some native attribute to edit their values, when needed.

For instance :
```
export const VtmnLink = ({
   // no "rel" nor "target" props here 
  ...,
  // if user add native rel or target attributes, they will be stored in the following:
  ...props
}: VtmnLinkProps) => {

  // "Intercept" attributes to edit it
  const relAttrValues = Array.from(
    new Set([
      // If component has a given "rel" attribute, keep values
      ...(props?.rel?.split(' ') ?? []),
      // If comp has "target" set to "blank", add some values
      ...(props?.target === '_blank' ? ['noopener', 'noreferrer'] : []),
    ]),
  ).join(' ').trim();

  return (
    <a
      ...
      {...props}
      {...(relAttrValues ? { rel: relAttrValues } : {})}
    >
      ...
    </a>
  );
};
```